ci/testsettings.py

```diff
@@ -1,3 +1,7 @@
+import os
+
+MAJOR_SOLR_VERSION = int(os.environ.get('SOLR_VERSION', '8').split('.')[0])
+configset = 'basic_configs' if MAJOR_SOLR_VERSION < 7 else '_default'
```

Comment:
I think this needs to be in parasolr code somewhere and not just in our test settings — aren't there are a few places besides tests where we need to know the default?

Reply:
Or does it not matter because we're assuming solr 8, and applications that need to should override if necessary? (that sounds great)

Reply:
From conversation during pairing: Reintroduce the default configset in non-test code.